website/docs/r/online_archive.html.markdown

```diff
@@ -25,15 +25,20 @@ resource "mongodbatlas_online_archive" "test" {
     db_name      = var.database_name
 
     partition_fields {
-        field_name = "firstName"
+        field_name = "date_field"
         order = 0
     }
 
     partition_fields {
-        field_name = "lastName"
+        field_name = "firstName"
         order = 1
     }
 
+    partition_fields {
+        field_name = "lastName"
+        order = 2
+    }
+
     criteria {
         type = "DATE"
         date_field = "created"
```

Comment:
this field needs to be fixed. I run the terraform apply with this example and I got:

```
Error: error creating MongoDB Atlas Online Archive:: https://cloud-dev.mongodb.com/api/atlas/v2/groups/65f82c4dd4c91e6c61437623/clusters/cluster-test-3/onlineArchives POST: HTTP 400 Bad Request (Error code: "ONLINE_ARCHIVE_DATE_CRITERIA_NOT_IN_PARTITION") Detail: Date criteria is not specified in partition fields. Reason: Bad Request. Params: []
│
│   with mongodbatlas_online_archive.test,
│   on main.tf line 1, in resource "mongodbatlas_online_archive" "test":
│    1: resource "mongodbatlas_online_archive" "test" {
```
You should pick one of the three partition fields (e.g. date_field)

Reply:
now "dateField"